### PR TITLE
Automate execution of generateGuiClassesFromQtCreatorStudioUiFiles.py in dev environment

### PR DESCRIPTION
**Pull request by PuaYongHao**
Auto generate UI files #173

### CODE REVIEW

**Comment by martin-neugebauer** [comment]
I think it would be better if @zuckerruebe reviewed this.

File: README.md [hunk 1]
```diff
@@ -115,24 +115,25 @@ specified are for a Windows environment. They are very similar for Linux.
     ```commandline
     py -3.9 -m venv venv
     ```
-1. Activate it:
+2. Activate it:
```
Comment (zuckerruebe) [inline README.md:118]: Isn't it more convenient to let markdown "take care" of the numbering as before?

File: trnsysGUI/GUI.py [hunk 2]
```diff
@@ -739,4 +740,9 @@ def main():
 
 
 if __name__ == "__main__":
+    if pkg_resources.get_distribution("pytrnsys-gui").parsed_version.local.endswith('dev'):
```
Comment (zuckerruebe) [inline trnsysGUI/GUI.py:743]: Ah, so this works now? Great! What did we miss last time we talked about it?

Let's be generous and give this bit of code its own function with a meaningful name and maybe introduce some local variables on top? ;)

Comment (zuckerruebe) [reply trnsysGUI/GUI.py:743]: ...and then maybe call this function first thing in `main`?

**Review by zuckerruebe — CHANGES_REQUESTED** [review]
Other than that, great! (If it works, didn't check... ;) ).

File: README.md (hunk 1 above)
Comment (PuaYongHao) [reply README.md:118]: They changed by the system when I tried to add "Run `setup.py` with `dev` tag".

File: trnsysGUI/GUI.py (hunk 2 above)
Comment (PuaYongHao) [reply trnsysGUI/GUI.py:743]: Sure.  Last time we only had pkg_resources.get_distribution("pytrnsys-gui").endswith('dev') but it did not work.

Comment (zuckerruebe) [reply trnsysGUI/GUI.py:743]: Aha, I see.

**Pull request merged**